src/view/pages/Farmer/tabs/Buy/index.tsx

```diff
@@ -31,6 +31,13 @@ interface OwnStateType {
  * Buy Tab Component
  */
 class Buy extends React.Component<PropsType, OwnStateType> {
+
+  private onEntryPress = (uuid: string) => {
+    return () => {
+      // this.props.setActiveMilkEntry(uuid);
+      // this.props.navigateToMilkEntry();

```

Comment:
Why are these commented out?

Reply:
Ah I see now, maybe add a comment saying `follow this` or something, so they don't get confused.